### PR TITLE
Add selecting transfer action UI

### PR DESCRIPTION
**Pull request by duypateo**
- Adding new UI for Transfer flow, which is before selecting transfer account UI

### CODE REVIEW

**Comment by Git-on-my-level** [comment]
For UI PRs it's often useful to include screenshots or demo URLs, there's a script that lets you deploy a demo site to S3

**Comment by duypateo** [comment]
Screen Shot 2020-05-20 at 12 25 54 PM
Screen Shot 2020-05-20 at 12 26 13 PM
Screen Shot 2020-05-20 at 12 26 21 PM

File: ts/prime/Web/src/container/transfercations.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    console.log(this.props)
+    return (
+      <React.Fragment>
+        <Balance.Wrapper>
+          <Balance.Sidebar
+            availableBalance={this.state.availableBalance}
+            pendingBalance={this.state.pendingBalance}
+            totalBalance={this.state.totalBalance}
+          />
+          <TransferActionsComponent {...this.props} />
```
[the remark this answers is not in the document]
Comment (duypateo) [reply ts/prime/Web/src/container/transfercations.js]: Updated

**Comment by dhanesh-kapadiya** [comment]
Closing because changes are already merged.

**Pull request closed without merging**